props/saber_fett263_buttons.h

```diff
@@ -685,6 +685,10 @@ CUSTOM SOUNDS SUPPORTED (add to font to enable):
 #define FETT263_USE_SETTINGS_MENU
 #endif
 
+#if defined(FETT263_SAY_COLOR_LIST) || defined(FETT263_SAY_COLOR_LIST_CC)
+#define SAY_COLOR_LIST
```

Comment:
unrelated?

Reply:
it's an additional memory saving for sound_library when SAY_COLOR_LIST defines are not enabled.  So tangentially related?